ci-operator/config/openshift/cluster-storage-operator/openshift-cluster-storage-operator-master.yaml

```diff
@@ -92,6 +92,13 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-e2e-azure-csi
+- as: e2e-aws-shared-resources
+  optional: true
```

Comment:
```suggestion
  optional: true
  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```

Reply:
I don't think that adding it only to shared-resources is helpful. It should be either in (almost?) all CI jobs or in none.

Reply:
I will update them in a subsequent pull request for all jobs, thanks!